### DISCUSSION TITLE
设置分享：插件新版本下的 Switch 游戏联机配置

### DISCUSSION THREAD

**Discussion by vernesong** [vernesong/OpenClash#3684] [Show and tell]
# 视网络环境不同效果可能有差异，请自行测试，作者本地网络环境下可稳定联机

### 1、模式及网络栈
![image](https://github.com/vernesong/OpenClash/assets/42875168/bbc01f6f-42f0-4feb-95bd-888ad52023a3)

### 2、fake-ip-filter
![image](https://github.com/vernesong/OpenClash/assets/42875168/9e6dd884-2a2e-409b-9023-22efe76cdd84)

### 3、自定义规则（为switch单独设置策略组）
![image](https://github.com/vernesong/OpenClash/assets/42875168/7f6d82e2-862a-4591-8f4f-486724ab44cc)

### 4、检查UTUN接口状态
![image](https://github.com/vernesong/OpenClash/assets/42875168/662b0518-ada0-42a9-8753-878b25ed352f)

### 5、控制面板选择节点（请确保节点稳定性且支持UDP）
![image](https://github.com/vernesong/OpenClash/assets/42875168/41b15e74-5e56-4b8d-876c-8d1c7fe1cc86)

### 6、检查Switch连接是否符合策略组及规则设定
![image](https://github.com/vernesong/OpenClash/assets/42875168/8281764e-9382-4ddc-9481-50f3a0a6059d)

### 1、设备网络测试
![image](https://github.com/vernesong/OpenClash/assets/42875168/aeb66247-8210-4450-82ea-c3ef779555be)

### 2、设备网络概览（插件运行在旁路由）
![image](https://github.com/vernesong/OpenClash/assets/42875168/76832c93-0ea9-4130-a774-f063dbac210c)

### 3、斯普拉遁3 联机测试
![image](https://github.com/vernesong/OpenClash/assets/42875168/f53f9218-3ce3-4573-837c-eaeafe8a8096)

### 4、马里奥赛车8 联机测试
![image](https://github.com/vernesong/OpenClash/assets/42875168/d302f6c3-7d49-4dc2-bcad-bea13f9cef11)
![image](https://github.com/vernesong/OpenClash/assets/42875168/53eba757-f0b7-4f92-8ea0-4e9e5a33b709)

**Comment by franklee628** [comment]
照着教程设置了以后，Xbox可以稳定NAT Open，Switch的NAT在A-D之间变来变去。

**Comment by bibibivlin** [comment]
用openClash后Switch进行设置里的测试永远都是A和满速，但是在游戏里面永远都是联机失败
直连下NAT是B但是至少游戏里能连上对局（不过有时候掉线……

**Reply by jacobsenxin** [reply]
跟你一样的问题？请问您解决了吗？

**Reply by BedrockDigger** [reply]
我用`fake-ip`也是类似情况，喷喷能匹配但进局之后经常10秒内就连接错误。换到`redir-host（混合）`模式能玩了，但现在`redir-host`没法日用...

**Reply by BedrockDigger** [reply]
按理说目前meta内核的`redir-host`就是`fake-ip-filter`为`*`的`fake-ip`，可能是`fake-ip-filter`漏加了域名？

**Comment by vernesong** [comment]
那是你的服务器有问题

**Reply by jacobsenxin** [reply]
大佬，新版本没有自动创建UTUN接口，有教程可以指个路吗？谢谢了
